recipes/gz-sim-yarp-plugins/meta.yaml

```diff
@@ -70,13 +70,12 @@ outputs:
         {% endfor %}
 
   - name: {{ plugin_name }}
-    requirements:
-      run:
-        - {{ pin_subpackage(cxx_name, exact=True) }}
     test:
       commands:
-        # - test -f ${PREFIX}/include/GazeboYarpPlugins/common.h  # [not win]
-        # - if not exist %LIBRARY_INC%\\GazeboYarpPlugins\\common.h exit 1  # [win]
+        {% for gyp_lib in gyp_libs %}
```

Comment:
I am not sure if `gyp_libs` variable is visibile here.

Reply:
Me too, let's see the CI logs